docs/install/_obsolete_install-server.md

```diff
@@ -221,4 +221,4 @@ docker exec -it isle-apache-ld bash /utility-scripts/isle_drupal_build_tools/isl
 ---
 
 ## Migrate to ISLE Environment 
-* Please click [Migrate to ISLE Environment](../migrate/install-migration.md) if you are installing ISLE to migrate your existing production Islandora 7.x environment to an ISLE environment. This documentation will help you identify and copy your institution's preexisting Islandora data, files, and themes (including your data volume, Drupal site(s) and theme(s), and commonly customized xml and xslt files) to your ISLE environment.
+* Please click [Migrate to ISLE Environment]() if you are installing ISLE to migrate your existing production Islandora 7.x environment to an ISLE environment. This documentation will help you identify and copy your institution's preexisting Islandora data, files, and themes (including your data volume, Drupal site(s) and theme(s), and commonly customized xml and xslt files) to your ISLE environment.
```

Comment:
[Migrate to ISLE Environment]()  Missing URL?

Reply:
actually, I removed the link because the file is now missing. That file has been split into 3 migration files (local, staging, production). I'd like to see this removed, hence my renaming it to be `_obsolete_install-server.md`.